packages/opentelemetry-metrics/src/MeterProvider.ts

```diff
@@ -61,22 +63,38 @@ export class MeterProvider implements api.MeterProvider {
     return this._meters.get(key)!;
   }
 
-  shutdown(cb: () => void = () => {}): void {
-    this._shutdownAllMeters().then(() => {
-      setTimeout(cb, 0);
-    });
+  shutdown(): Promise<void> {
     if (this._cleanNotifyOnGlobalShutdown) {
       this._cleanNotifyOnGlobalShutdown();
       this._cleanNotifyOnGlobalShutdown = undefined;
     }
+    return this._shutdownAllMeters();
   }
 
-  private _shutdownAllMeters() {
-    if (this._config.exporter) {
-      this._config.exporter.shutdown();
+  private _shutdownAllMeters(): Promise<void> {
```

Comment:
async?

Reply:
here the same it return a promise so that you can call it many times, as the exporter is being shutdown too